.github/workflows/release.yml

```diff
@@ -0,0 +1,24 @@
+name: Release
+
+on:
+  push:
+    tags:
+      # Publish on any tag starting with a `v`, e.g. v1.2.3
+      - v*
+
+jobs:
+  pypi:
+    name: Publish to PyPI
+    runs-on: ubuntu-latest
+    # Environment and permissions trusted publishing.
+    environment:
+      name: production
+
+    permissions:
+      id-token: write
+    steps:
+      - uses: actions/checkout@v4
```

Comment:
Wow the release pipeline is so much simpler now

Reply:
And super fast... uv and ruff are magical :D